mm2src/coins/init_create_account.rs

```diff
@@ -28,8 +28,23 @@ pub struct CreateNewAccountRequest {
     params: CreateNewAccountParams,
 }
 
+#[derive(Deserialize)]
+#[serde(rename_all = "snake_case")]
+pub enum CreateAccountScanPolicy {
```

Comment:
Do you expect this enum to have more variants in the future? If no, is it worth using bool instead? :slightly_smiling_face:

Reply:
`CreateAccountScanPolicy` has similar variants as `EnableCoinScanPolicy`:
https://github.com/KomodoPlatform/atomicDEX-API/blob/hd-account-storage/mm2src/coins/coin_balance.rs#L201-L209

I thought it's more clear for the user if he is allowed to use same `DoNotScan` and `Scan` policies. But I agree it's easier to use bool.
Will replace `CreateAccountScanPolicy` with bool.

Reply:
Is this struct not required anymore?